### PR TITLE
grafana/toolkit: expose Jest maxWorkers arg for plugin test & build tasks

### PR DESCRIPTION
**Pull request by domasx2**
**What this PR does / why we need it**:
By default Jest spawns  num logical cores - 1 amount of workers for running tests. Tests that use react testing library can consume a lot of memory, so we're having memory issues when running plugin builds in CI, or even on own machine with high core count. 
This PR exposes jest's `maxWorkers` arg for toolkit's cli tasks, so number of Jest workers can be limited:
```sh
./node_modules/.bin/grafana-toolkit plugin:test --maxWorkers=2
./node_modules/.bin/grafana-toolkit plugin:ci-build --maxJestWorkers=2
./node_modules/.bin/grafana-toolkit plugin:build --maxJestWorkers=2
```

### CODE REVIEW

File: packages/grafana-toolkit/src/cli/index.ts [hunk 1]
```diff
@@ -138,9 +138,10 @@ export const run = (includeInternalScripts = false) => {
 
   program
     .command('plugin:build')
+    .option('--maxJestWorkers <num>|<string>', 'Limit number of Jest workers spawned')
     .description('Prepares plugin dist package')
     .action(async cmd => {
-      await execTask(pluginBuildTask)({ coverage: false, silent: true });
+      await execTask(pluginBuildTask)({ coverage: false, silent: true, maxJestWorkers: cmd.maxJestWorkers });
```
Comment (hugohaggmark) [inline packages/grafana-toolkit/src/cli/index.ts:144]: Out of curiosity what happens if `cmd.maxJestWorkers` or `cmd.maxWorkers` is undefined in all these cases, do we need a default or is this just going to work?

Comment (domasx2) [reply packages/grafana-toolkit/src/cli/index.ts:144]: It works like before then, with Jest defaulting to ncpu - 1

Comment (hugohaggmark) [reply packages/grafana-toolkit/src/cli/index.ts:144]: Ok thank you

**Review by dprokop — APPROVED** [review]

**Review by dprokop — COMMENTED** [review]
Thanks @domasx2, this is great! I think this needs to be added to toolkit changelog, so I'm adding correct labels and assigning 7.3 milestone

**Review by ryantxu — APPROVED** [review]
LGTM

**Comment by torkelo** [comment]
Can you merge this with master and resolve conflicts?

**Comment by domasx2** [comment]
> Can you merge this with master and resolve conflicts?

done!

**Comment by torkelo** [comment]
Thanks for contributing!

**Pull request merged**